README.md

```diff
@@ -39,14 +36,43 @@ Steps to run in locally in development mode:
 
 `go run . dot --serverConfig=rhn.conf |  dot -Tx11`
 
-### Profile
-Run with profile: `go run . -cpuprofile=cpu.prof -memprofile=mem.prof ...`
+## Build and release
+
+### 1. Update cmd version
+
+- Edit file `cmd/root.go` "Version" property to the desire version
+- On project root folder run `osc vc` to update the changes file with the release data
+- Manually update changes file with the release number for hte next release
```

Comment:
```suggestion
- Manually update changes file with the release number for the next release
```
What about the version at the SPEC?

Reply:
that will be updated by the _service on OBS. I will change the version on spec file to a meanless one